apps/astarte_housekeeping/lib/astarte_housekeeping/queries.ex

```diff
@@ -44,13 +45,16 @@ defmodule Astarte.Housekeeping.Queries do
 
   def create_realm(realm_name, public_key_pem, replication, device_limit, max_retention, opts) do
     with :ok <- validate_realm_name(realm_name),
+         keyspace_name <-
+           CQLUtils.realm_name_to_keyspace_name(realm_name, Config.astarte_instance_id!()),
```

Comment:
Nit: in a `with` block, `<-` is used for matching, but here you're just binding the variabile (ok, matching is the way Elixir does binding, but...). Be more explicit using
```suggestion
         keyspace_name =
           CQLUtils.realm_name_to_keyspace_name(realm_name, Config.astarte_instance_id!()),
```